### PR TITLE
Classification Metrics: Parameters Normalization

### PR DESCRIPTION
**Pull request by oualib**
- multilabel_confusion_matrix is a wrong name for what it is. We've built one unified confusion_matrix function
 - Parameter scores is replaced to None as the one of sklearn.

TODO:
 - add the binary option.

### CODE REVIEW

**Review by vikash018 — COMMENTED** [review]
Looks good.

**Review by vikash018 — APPROVED** [review]

**Review by vikash018 — APPROVED** [review]
